spec/classes/jenkins_slave_spec.rb

```diff
@@ -67,6 +68,19 @@
     end
   end
 
+#  describe 'Darwin' do
```

Comment:
I'm still learning rspec, and can't figure out how to get the tests to ignore java install on darwin.

```
Failures:

  1) jenkins::slave Darwin with slave_name behaves like using slave_name should contain File[/home/jenkins-slave/jenkins-slave] with content =~ /^CLIENT_NAME="jenkins-slave"$/
     Failure/Error: it { should contain_file(slave_runtime_file).with_content(/^CLIENT_NAME="jenkins-slave"$/) }
     Puppet::Error:
       Could not find resource 'Class[Java]' for relationship on 'Service[jenkins-slave]' on node myosx.local
     Shared Example Group: "using slave_name" called from ./spec/classes/jenkins_slave_spec.rb:80
     # ./spec/classes/jenkins_slave_spec.rb:43:in `block (3 levels) in <top (required)>'
```